### ISSUE TITLE
GL Validation

### ISSUE THREAD

**Issue by koushik-ac** [beezlabs-org/automation-dev#121]

**Comment by mbharathm** [comment]
While going through the docs, I see this as an extension to previous Payroll ADP. It looks like this time they wanted to automated the source extraction part also. Do you align with this from the documentation @koushik-ac . We shall ask for a call and ask this.
We should take this opportunity in proposing the roll out to other pending markets they kept on hold, but we shall go with their ask first and understand the need clearly.

**Comment by koushik-ac** [comment]
Call has been scheduled for Monday to complete the Initial Clarification.

**Comment by koushik-ac** [comment]
The automation requirement is related to Payroll ADP Automation. We have requested the Op Ex team to get in touch with Stephanie Nuñez and try to understand the overall scope of automation whether it is expected to create a Cockpit Proposal or just perform the validation or send the results.

**Comment by mbharathm** [comment]
Followed up with OpEx today to setup a call with end users. Sharon to look for the next available for all stakeholders and setup one this week or early next week.

Meanwhile it was understood from Consultants that Vivek has started analysing the use case steps received on the validation part.

**Comment by mbharathm** [comment]
Call has been setup with End users on 13-Nov 9:30 PM IST 
Cc: @koushik-ac

**Comment by koushik-ac** [comment]
@vivekmishra9098 - He will be providing a list of Clarifications by EOD today and the same will be shared with the end users before the call.

**Comment by mbharathm** [comment]
@koushik-ac @vivekmishra9098  , please update the ETA by when you will have the analysis ready.

**Comment by Dean13b** [comment]
Next actions planned for this week from Beezlabs

**Comment by mbharathm** [comment]
@koushik-ac  As per our discussion on call yesterday, our next steps is to prepare the PDD and have an alignment call on the scope/PDD validation from end user, Juan and Karla as there is a hold on two markets for later stage. This needs to be aligned in advance with Juan&Karla before commencing with development. On BES call with Karla on Thursday, this has to be brought to discussion FYI.

Let's have the PDD walk-through call this week, I will let you suggest a date by today.

Cc: @dominicceo

**Comment by koushik-ac** [comment]
Yes @mbharathm. We shall have a PDD walk-through call on Friday(02/02/2024) of this week.

**Comment by mbharathm** [comment]
Meeting for PDD walkthrough is setup for suggested date @koushik-ac . To make it more productive, let's plan to have the PDD more on functional side with test sceanrios (like how SAQ ASN use case was executed by Sakthi/Gowtham). It gives us more comfort to us and end users in not missing things at a later stage.

**Comment by koushik-ac** [comment]
PDD Walkthrough call has been setup for Monday. Once the PDD is approved, we shall start the development. This is a Medium Complex Use Case and we foresee three weeks of development effort for this.
